inc/woocommerce.php

```diff
@@ -16,7 +16,18 @@
  * @return void
  */
 function _s_woocommerce_setup() {
```

Comment:
I couldn't comment on the `add_action` line, but just a small comment: would it be good to wrap the `add_action( 'after_setup_theme', '_s_woocommerce_setup' )` in a conditional like:

```php
if ( class_exists( 'WooCommerce' ) ) {
    add_action( 'after_setup_theme', '_s_woocommerce_setup' )
}
```

So that this function is added only in the case the `WooCommerce` plugin is installed and active?

Reply:
That is not necessary as it is already being done on a file level. See: https://github.com/Automattic/_s/blob/master/functions.php#L165

Reply:
Oh, didn't notice this. Great :+1: